routers/graphql-router.js

```diff
@@ -15,7 +15,9 @@ dbConnector.connect().then(() => {
     const userCollection = new MongoDBCollection(dbConnector.client, DATABASE_NAME, USER_COLLECTION);
     const dataInterface = new User(userCollection);
     root = {
-        getMyUser : dataInterface.getMyUser.bind(dataInterface)
+        getMyUser : dataInterface.getMyUser.bind(dataInterface),
+        grantToken: dataInterface.grantToken.bind(dataInterface),
+
```

Comment:
the invalidateToken API call needs to be implemented as part of this change as well